.travis/script.sh

```diff
@@ -4,9 +4,9 @@ set -x
 # If this is a master build then deploy at the end
 if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
   mvn -Dstyle.color=always -B -U -P sign --settings travis-settings.xml deploy &&
-  # TODO - after testing this, make it only publish on a release build
-  mvn site site:stage post-site scm-publish:publish-scm;
+  mvn site site:stage post-site;
```

Comment:
i've never seen people run mvn site on the same line as mvn deploy, so have left it as two incantations